src/views/Profile.vue

```diff
@@ -1,13 +1,369 @@
 <template>
-  <div></div>
+  <div class="mx-auto py-10">
+    <div class="flex justify-between">
+      <h1 class="text-6xl">{{ $t('profile.title') }}</h1>
+    </div>
+
+    <div class="px-4 py-12 sm:py-16 my-10 bg-white shadow rounded-lg sm:px-10">
+      <div class="max-w-5xl mx-auto h-auto">
+        <div class="grid grid-cols-1 sm:grid-cols-4 gap-y-8">
+          <div class="relative w-full col-span-1 sm:col-span-3">
+            <span class="block">Email: {{ user.email }}</span>
+            <span class="block">Created at: {{ getCreatedAt }}</span>
+            <span class="block">Last updated: {{ getUpdatedAt }}</span>
+
+            <div v-if="edit" class="relative flex sm:absolute -bottom-4 text-3xl mt-0 sm:mt-5 w-full sm:w-5/6">
+              <base-input class="w-full sm:w-1/2 mr-1" v-model="profile.first_name" :label="$t('register.name.firstName')" />
+              <base-input class="w-full sm:w-1/2" v-model="profile.last_name" :label="$t('register.name.lastName')" />
+            </div>
+            <div v-else class="relative sm:absolute bottom-0 text-3xl mt-5">
+              <span>{{ user.first_name }} {{ user.last_name }}</span>
+            </div>
+          </div>
+          <div class="flex flex-col">
+            <img class="rounded-full h-32 w-32 object-cover mx-auto" :src="getAvatar" :alt="$t('profile.avatar.alt')">
+            <span v-if="!edit" class="flex justify-center mt-2 sm:mt-0">
+              <base-button outline size="sm" @click="edit = !edit">
+                {{ $t('profile.editProfile.title') }}
+              </base-button>
+            </span>
+            <span v-else class="flex justify-center mt-2 sm:mt-0">
+              <base-button class="w-1/2 mr-1" color="secondary" outline size="sm" @click="showModal = true">{{ $t('profile.changeAvatar.title') }}</base-button>
+              <base-button class="w-1/2" color="danger" outline size="sm" @click="onClear">{{ $t('profile.clearAvatar.title') }}</base-button>
+            </span>
+          </div>
+        </div>
+        <div v-if="edit" class="sm:min-w-0 mt-6 inline-flex flex-row-reverse w-full">
+          <base-button outline class="mr ml-1" size="md" @click="onSave">
+            {{ $t('profile.editProfile.save') }}
+          </base-button>
+          <base-button class="" size="md" @click="edit = !edit">
+            {{ $t('profile.editProfile.cancel') }}
+          </base-button>
+        </div>
+      </div>
+    </div>
+
+    <transition name="fade">
```

Comment:
Please extract everything related to avatar upload into a separate `AvatarUpload` component